.github/workflows/pythonapp.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip3 install -r requirements.txt
```

Comment:
I think you may have to run your install script here. Might be why the tests are failing.